tests/test_long_weekends.py

```
@@ -0,0 +1,113 @@
+        with self.assertRaises(ValueError):
+            find_long_weekends("IN", 2025, month=13)
+
+    def test_no_holidays(self):
+        long_weekends = find_long_weekends("IN", 2025, month=2)
+        self.assertEqual(long_weekends, [])
+
+    def test_leap_year(self):
+        long_weekends = find_long_weekends("IN", 2024)  # 2024 is a leap year
+        self.assertIsInstance(long_weekends, list)
+
+def find_long_weekends(country, year, month=None, language=None, exact_range=False):
```

Ruff (0.8.2)

Redefinition of unused `find_long_weekends` from line 15

(F811)